lib/libzfs/libzfs_pool.c

```diff
@@ -2185,7 +2188,7 @@ zpool_find_vdev(zpool_handle_t *zhp, const char *path, boolean_t *avail_spare,
 }
 
 static int
-vdev_online(nvlist_t *nv)
+libzfs_vdev_online(nvlist_t *nv)
```

Comment:
Name conflict, triggered by adding mmp_update_uberblock() and calling it from vdev_config_sync().